dev/source/docs/SITL_simulation_parameters.rst

```diff
@@ -0,0 +1,192 @@
+
+Many of the following tasks involve setting parameter values over
+MAVLink, which you do using the ``param set`` command as shown:
+
+::
+
+    param set PARAMETERNAME VALUE
+
+All available parameters can be listed using ``param show``. The
+SITL-specific parameters start with ``SIM_``, and can be obtained using:
+
+::
+
+    param show SIM_*
+
+.. tip::
+
+   A table of user settable SITL-specific parameters can be found in `SITL.cpp <https://github.com/ArduPilot/ardupilot/blob/master/libraries/SITL/SITL.cpp>`__
+   When you change a parameter the value remains in the virtual EEPROM
+   after you restart SITL. Remember to change it back if you don't want it
+   any more (or :ref:`reload/reset the parameters <using-sitl-for-ardupilot-testing_loading_a_parameter_set>`). 
+
+
+Changing the speed of the simulation
+------------------------------------
+
+Most of the simulator backends (native, RealFlight,etc) support changing the speed while
+running (NOT Realflight). Just set the ``SIM_SPEEDUP`` parameter as needed. A value of 1
+means normal wall-clock time. A value of 5 means 5x realtime. A value
+of 0.1 means 1/10th of real time.
+
+Running with high-levels of speedup can result in enough network traffic that MAVProxy can’t keep up. An error such as “Set RC override timeout” or the vehicle entering GCS failsafe are typical of these failures. Reducing the ``SIM_SPEEDUP`` factor is typically sufficient to avoid this problem. Also, using the ``--udp`` options will use UDP instead of the slower TCP communication protocol and can help avoid this issue at higher speedups.
+
+Loading a parameter set
+-----------------------
+
+When starting SITL the first time, the device will be configured with
+default parameters for the vehicle/frame type. Typically, you will want to replace these with
+values that simulate the desired vehicle and environment conditions.
+
+::
+
+    param load <<path to parameter file>
+
+the working directory is the directory in which MAVProxy starts, not the simulation
+
+You can re-load the parameters later if you choose, or revert to the
+default parameters by starting SITL (**sim_vehicle.py**) with the
+``-w`` flag.
+
+.. note:: You may have to load them again, after typing ``param fetch``, in order to load parameters that require enabling before presenting their parameter set.  And, in some cases, you may even need to restart SITL in order for some new parameters, such as output function changes, to take effect. This can be avoided if you add the parameter file during the start of SITL with the "--add-param-file=*pathtofile* "....ie: sim_vehicle.py -f flightaxis:192.168.x.x - -map - -console --add-param-file=*pathtofile* -w. This adds the param file as a default and then wipes any previous param changes that may exist in the simulation directory. 
+
+Parameters can also be saved. For example, to save the parameters into
+the present working directory you might do:
+
+::
+
+    param save ./myparams.parm
```

Comment:
Perhaps mention `param help` for other options?

Reply:
added